neqo-transport/src/connection.rs

```diff
@@ -245,11 +301,20 @@ impl Connection {
         tps.set_empty(tp_const::DISABLE_MIGRATION);
     }
 
-    fn new<A: ToString, I: IntoIterator<Item = A>>(
+    pub(crate) fn original_connection_id(&mut self, odcid: &ConnectionId) {
```

Comment:
Depending on the outcome of Retry discussions, it might make sense to move this into the constructor.  A private constructor that included the ODCID would be a potential path forward.  We need to ensure that the server creates epoch 0 keys at the right time and that it doesn't end up creating them twice if it doesn't need to.